### PR TITLE
feat(Card): Add Card component to storybook

### PR DESCRIPTION
**Pull request by Artikodin**
- [x] Feature
- [ ] Fix
- [ ] Enhancement

## Description

The ticket [BP-266](https://tillersystems.atlassian.net/jira/software/projects/BP/boards/57/backlog?selectedIssue=BP-266) ask us to create a pie chart for Sonar dashboard. The pie chart is display in a card which will be generic and reused in all Sonar page. As @sun-tea  [comment](https://github.com/tillersystems/tiller-sonar/pull/1114#pullrequestreview-269118747) says, we should put that card in Stardust for reusability.

## Todo - Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.

### CODE REVIEW

File: src/Card/__stories__/Card.stories.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  })
+  .add('with properties', () => {
+    const isHeader = boolean('With Header', true, 'Layout');
+    const isFooter = boolean('With Footer', true, 'Layout');
+    return (
+      <div>
+        <Card height="350px" width="350px">
+          {isHeader && (
+            <Card.Header>
+              <h3 style={{ padding: '1.6rem 2rem' }}>Header</h3>
```
Comment (sun-tea) [inline src/Card/__stories__/Card.stories.js]: I think the style could be directly in the Card.Header, see my comment on the component

File: src/Card/Header/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+/**
+ * Default props
+ */
+Header.defaultProps = {
+  children: null,
+  className: '',
+};
+
+export default styled(Header)`
+  border-bottom: solid 1px ${({ theme: { palette } }) => palette.lightGrey};
```
Comment (sun-tea) [inline src/Card/Header/index.js]: Depending on a string prop `size`, the Header could put a different style:

```
display: flex;
  align-items: center;

  height: 7.4rem;
  padding: 0 3rem;

  ${({ size }) =>
    size === 'small' &&
    css`
      height: 6rem;
      padding: 0 2rem;
    `}
```

and so you can remove the padding in your example from storybook. Same goes for the Footer.

Comment (sun-tea) [inline src/Card/Header/index.js]: The color of borders is `veryLightBlue` (cf design). We also need to change that in Sonar

File: src/Card/Footer/index.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+/**
+ * Default props
+ */
+Footer.defaultProps = {
+  children: null,
+  className: '',
+};
+
+export default styled(Footer)`
+  box-shadow: 0 -1px 2px 0 rgba(0, 0, 0, 0.04);
```
Comment (sun-tea) [inline src/Card/Footer/index.js]: Remove the box-shadow

**Comment by sun-tea** [comment]
I wonder: should we name that component `Tile` instead? Since it goes with the grid concept

**Comment by Artikodin** [comment]
> I wonder: should we name that component `Tile` instead? Since it goes with the grid concept

For me a `Tile` is a piece of the grid like a cell for an array. That component is a Card, which in most of the case gonna be displayed in a Grid to fit to the design specification. But we should consider the fact that it can be used outside of a grid layout without problem.

What do you think ? @spartDev @sun-tea

**Comment by sun-tea** [comment]
I'll let @spartDev decide about the naming

**Pull request merged**